prov/shm/src/smr_progress.c

```diff
@@ -110,19 +111,23 @@ static int smr_progress_resp_entry(struct smr_ep *ep, struct smr_resp *resp,
 			break;
 		if (pending->cmd.msg.hdr.op == ofi_op_read_req) {
 			if (!*err) {
-				pending->bytes_done =
+				hmem_copy_len =
```

Comment:
I think the variable name should be `ret`, it is not always length.

Reply:
I didn't use ret because that already exists in some functions and I was trying to maintain uniformity. I didn't just use ret where it already existed because it is an int and we need an ssize_t. Would it make sense to rename `hmem_copy_len` to `hmem_copy_ret`?

Reply:
`hmem_copy_ret` is OK to me.